azurelinuxagent/common/AgentGlobals.py

```diff
@@ -17,6 +17,60 @@
 # Requires Python 2.6+ and Openssl 1.0+
 
 
+# too-few-public-methods<R0903> Disabled: This class is used as an Enum
+class FeatureNames(object):  # pylint: disable=R0903
+    """
+    Enum for defining the Feature Names for all internal and CRP features
+    """
+    MultiConfig = "MultipleExtensionsPerHandler"
+    ExtensionTelemetryPipeline = "ExtensionTelemetryPipeline"
```

Comment:
Let's also separate ETP from this list of features. I think the intention of this list was to be used for the features that require interaction between the agent and CRP and ETP doesn't quite fit there

Reply:
I mainly added it here to have a central place for all features, but agreed its not exactly the perfect place for it. Will make the changes

Reply:
Removed ETP